docs/ROADMAP.md

```diff
@@ -4,15 +4,15 @@ Last updated: September 14th, 2021
 
 Draft of *v1* is released. Please follow the [installation](https://kpt.dev/installation/) guide.
 
-### 1. Declarative function pipeline
+### Declarative function pipeline
```

Comment:
Let's move all the delivered things in order (by when they were delivered) at the bottom.  

The sections should be
1) what's coming up next in chronological order
2) Historical roadmap

Reply:
Updated.